app/components/Resources/ResourcesTable.js

```diff
@@ -98,16 +101,16 @@ class ResourcesTable extends Component {
 		let page = 0;
 		if(this.state.openFilter) {
 			this.setState({
-				page: page,
+				page,
 				resources: this.state.allResources,
-				currentPage: this.state.allResources.slice(page, page + 9),
+				currentPage: this.state.allResources.slice(page, resultsPerPage),
```

Comment:
page is explicitly reset to 0 at the beginning of the function. I changed it, but it seems like a lateral change to me.